libs/wingsdk/src/ex/react-app.ts

```diff
@@ -0,0 +1,156 @@
+import { existsSync } from "fs";
+import { isAbsolute, join, resolve } from "path";
+import { Construct } from "constructs";
+import { WebsiteOptions, IWebsite } from "../cloud/website";
+import { fqnForType } from "../constants";
+import { App } from "../core";
+import { Resource, Node } from "../std";
+
+const DEFAULT_BUILD_FOLDER = "/build";
+export const DEFAULT_REACT_APP_BUILD_COMMAND = "npm run build";
+const DEFAULT_PORT = 3001;
+/**
+ * The file to write the wing environment to
+ */
+export const WING_JS = "wing.js";
+
+/**
+ * Global identifier for `ReactApp`.
+ */
+export const REACT_APP_FQN = fqnForType("ex.ReactApp");
+
+/**
+ * Options for `ReactApp`.
+ */
+export interface ReactAppProps extends WebsiteOptions {
+  /**
+   * The path to the React app root folder- can be absolute or relative to the wing folder
+   */
+  readonly projectPath: string;
+  /**
+   * The path to the React app build folder- relative to the `projectPath`
+   * @default "/build"
+   */
+  readonly buildDir?: string;
+  /**
+   * A command for starting React app locally
+   * @default "npm run start"
+   */
+  readonly startCommand?: string;
+  /**
+   * A command for building the React app
+   * @default "npm run build"
+   */
+  readonly buildCommand?: string;
+  /**
+   * In sim, if `true` - will use the start command, and if `false` - the build command
+   * @default true
+   */
+  readonly isDevRun?: boolean;
```

Comment:
I know it came up before, but this property still feels a bit strange. Let's try to discuss and find a better name for it or a better docstring

Reply:
switched to `useBuildCommand`